### ISSUE TITLE
Views: Create tests for main_screen.dart

### ISSUE THREAD

**Issue by palisadoes** [PalisadoesFoundation/talawa#1225]
The Talawa code base needs to be 100% reliable. This means we need to have 100% test code coverage.

Tests need to be written for file `lib/views/main_screen.dart`

- When complete, all methods, classes and/or functions in the file will need to be tested. 
- These tests must be placed in a single file with the name `test/views_test/main_screen_test.dart`. You may need to create the appropriate directory structure to do this.

### IMPORTANT:
Please refer to the parent issue on how to implement these tests correctly:

- https://github.com/PalisadoesFoundation/talawa/issues/1217

### PR Acceptance Criteria

- When complete this file must show **100%** coverage when merged into the code base. This will be clearly visible when you submit your PR.
- [The current code coverage for the file can be found here](https://codecov.io/gh/PalisadoesFoundation/talawa/tree/develop/lib/views/). If the file isn't found in this directory, or there is a 404 error, then tests have not been created.
- The PR will show a report for the code coverage for the file you have added. You can use that as a guide.
- You can verify your own code coverage by creating an account at [Codecov.io](https://codecov.io)

**Comment by Shivxnshjasathi** [comment]
Sir, can you assign this issue to me?

**Comment by noman2002** [comment]
Unassigned due to inactivity.

**Comment by literalEval** [comment]
I would like to work on this issue. Please assign this to me.

**Comment by literalEval** [comment]
@noman2002 I am facing some issues while writing test for this file. Can you help ?

**Comment by literalEval** [comment]
@noman2002 @CyberWake while writing tests for this file, I am getting `Multiple widgets used the same GlobalKey` error. Some of it was with the pluginList, which I fixed by renaming the duplicate keys, but the last error comes from `explore_events.dart` file, in which the `homeModel?.keySEAdd` key is used for FAB only once, but still throws this error.

Can you please help in resolving this ?
